qml/Silica4v-play/ApplicationWindow.qml

```diff
@@ -24,8 +24,18 @@ App {
         Theme.colors.statusBarStyle = Theme.colors.statusBarStyleWhite
     }
 
+    FontLoader {
+        id: material
+        source: "qrc:/icons/MaterialIcons-Regular.ttf"
+    }
+
     Component.onCompleted: {
         // We need to access the dp() function from the Theme component
         BVApp.Theme.myApp = app
+        if (material.status == FontLoader.Ready) {
```

Comment:
You can also leave it in. I just wanted to make sure that we don't have a potential race here.